Fix unexpected leading slash with filename in package.json

nolsherry opened the pull request:
PR for checking unexpected leading slash with filename in `filename` field,
and I have added this check item in `fixFormat.js`.
@pvnr0082t , @kennynaoh , please help me review this PR, thanks!

cc @PeterDaveHello , cc #9355

pvnr0082t commented on test/valid-packages-test.js:
```diff
@@ -320,6 +320,16 @@ packages.map(function(pkg) {
+    }
+  };
+
+  packageVows
```
What is this for....?

nolsherry replied:
Sorry, It is a mistake, I will remove `packageVows`.

kennynaoh commented on tools/fixFormat.js:
```diff
@@ -90,6 +90,15 @@ async.each(packages, function(item, callback) {
+      pkg.filename = filename;
+    }
+  }
+
```
Please remove blank line

nolsherry commented:
@pvnr0082t, I have modified the title of issue and fixed filename with leading slash of some libs in cdnjs.
@kennynaoh, I have removed the blank line, please help me review this PR, thanks！

pvnr0082t commented:
@nolsherry Any update here?

nolsherry commented:
Closed.
